operator/src/main/resources/scripts/startServer.sh

```diff
@@ -261,6 +264,11 @@ fi
 #
 
 createFolder "${DOMAIN_HOME}/servers/${SERVER_NAME}/security" "This is the server '${SERVER_NAME}' security directory within directory DOMAIN_HOME 'domain.spec.domainHome'." || exitOrLoop
+if [[ "${KUBERNETES_PLATFORM^^}" == "${PLATFORM_OPENSHIFT}" ]]; then
+  createFolder "${DOMAIN_HOME}/servers/${SERVER_NAME}/tmp" "This is the server '${SERVER_NAME}' tmp directory within directory DOMAIN_HOME 'domain.spec.domainHome'." || exitOrLoop
+  createFolder "${DOMAIN_HOME}/servers/${SERVER_NAME}/logs" "This is the server '${SERVER_NAME}' logs directory within directory DOMAIN_HOME 'domain.spec.domainHome'." || exitOrLoop
```

Comment:
This folder is already created in 'startNodeManager.sh' (see ' createFolder "${serverLogHome}"'), and its location varies based on LOG_HOME.

Reply:
But I suppose it might be helpful to create the directory anyway in case it's somehow used even when LOG_HOME is set.

Reply:
The tmp and logs directory is created when the server starts, somehow something down the subdirectory is not created with the right umask (group not writable),  so pre-creating it to make sure they are group writable.

Reply:
OK